pkgs/top-level/all-packages.nix

```diff
@@ -4037,8 +4039,6 @@ with pkgs;
 
   rdfind = callPackage ../tools/filesystems/rdfind { };
 
-  rdmd = callPackage ../development/compilers/rdmd { };
```

Comment:
Can you add an alias for that as well rdmd -> dtools in `./pkgs/top-level/aliases.nix`?